### DISCUSSION TITLE
Page reload breaks magewire routes

### DISCUSSION THREAD

**Discussion by nukanukanukanuka** [magewirephp/magewire#129] [General]
On the first page load everything is fine, I can adjust fields, submit the checkout form and it works correctly. Once I reload the checkout page, all magewire calls stop working, because the module is unable to find component block in layout. Magewire cache is enabled.

magento/vendor/hyva-themes/magento2-hyva-checkout/src/Model/Magewire/Component/Resolver/Checkout.php

$block = $page->getLayout()->getBlock($request->getFingerprint('name'));
if ($block === false) {
    throw new HttpException(404, 'Component could not be found.');
}

![image](https://github.com/magewirephp/magewire/assets/113438822/2eaa5dbe-d876-4e64-8316-9f6c8566c5f4)

**Comment by Vinai** [comment]
Hey Artur,
For Hyvä Checkout support questions please use the Hyvä Slack, as this repository is only about Magewire development. It actually is an independent project.
